interop-scoring/main.js

```diff
@@ -0,0 +1,522 @@
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/temporal-svc-encoding.https.any.html?h264',
+  '/webcodecs/temporal-svc-encoding.https.any.html?vp8',
+  '/webcodecs/temporal-svc-encoding.https.any.html?vp9',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?h264',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?vp8',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?vp9',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/full-cycle-test.https.any.html?av1',
+  '/webcodecs/full-cycle-test.https.any.html?h264_annexb',
+  '/webcodecs/full-cycle-test.https.any.html?h264_avc',
+  '/webcodecs/full-cycle-test.https.any.html?vp9_p0',
+  '/webcodecs/full-cycle-test.https.any.html?vp9_p2',
+  '/webcodecs/full-cycle-test.https.any.worker.html?av1',
+  '/webcodecs/full-cycle-test.https.any.worker.html?h264_annexb',
+  '/webcodecs/full-cycle-test.https.any.worker.html?h264_avc',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp9_p0',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp9_p2',
+  '/webcodecs/full-cycle-test.https.any.html?vp8',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp8',
+  // interop-2023-webcomponents
+  '/shadow-dom/focus/focus-shadowhost-display-none.html',
+  '/custom-elements/form-associated/ElementInternals-labels.html',
+  '/custom-elements/form-associated/ElementInternals-setFormValue.html',
+  '/custom-elements/form-associated/ElementInternals-validation.html',
+  '/custom-elements/form-associated/form-disabled-callback.html',
+]);
+
+
+// Calculate interop score (passing in all browsers) for a category
+// after tracking the category's scores for each browser.
+//
+// Subtests can be difficult to match accurately, as they can sometimes be missing
+// from runs entirely, and passing/failing cannot be discerned. The approach taken
+// here simply checks the percentage of passing subtests within a test
+// for each browser and adds the minimum percentage of passing subtests to the
+// interop score. Tests (not subtests) that are missing entirely from a
+// browser run are marked with an interop score of 0.
+//
+// This deviates from another approach that attempted to match each subtest result
+// together from all browser runs and calculate an interop score based on parallel
+// subtest results. Due to some missing subtests, this score skewed lower than the
+// current implementation. Neither is without its drawbacks, and the hope is that
+// the current approach will score runs more optimistically and avoid subtest matching.
+function aggregateInteropTestScores(testPassCounts, numBrowsers) {
+  if (testPassCounts.size === 0) return 0;
+  let aggregateScore = 0;
+  for (const testResults of testPassCounts.values()) {
+    let minSubtestPercentage = 1;
```

Comment:
This is a score 0-1, not a percentage, and it's the score for the test (based on subtest passes and totals), so I think this might be more accurate:

```suggestion
    let minTestScore = 1;
```